psi4/src/psi4/findif/findif.h

```diff
@@ -47,44 +48,46 @@ namespace findif {
 // functions to generate displacements
 std::vector< SharedMatrix > fd_geoms_1_0(std::shared_ptr<Molecule> mol, Options &options);
 // std::vector< SharedMatrix > fd_geoms_2_0(Options &options);
-std::vector< SharedMatrix > fd_geoms_freq_0(std::shared_ptr<Molecule> mol, Options &options, int irrep=-1);
-std::vector< SharedMatrix > fd_geoms_freq_1(std::shared_ptr<Molecule> mol, Options &options, int irrep=-1);
+std::vector< SharedMatrix > fd_geoms_freq_0(std::shared_ptr<Molecule> mol, Options &options, int irrep= -1);
+std::vector< SharedMatrix > fd_geoms_freq_1(std::shared_ptr<Molecule> mol, Options &options, int irrep= -1);
 std::vector< SharedMatrix > atomic_displacements(std::shared_ptr<Molecule> mol, Options &options);
 
 // functions to carry out finite-differences
 SharedMatrix fd_1_0(std::shared_ptr<Molecule> mol, Options &options, const py::list& energies);
 //PsiReturnType fd_2_0(std::shared_ptr<Molecule> mol, Options &options, const py::list& energies);
-SharedMatrix fd_freq_0(std::shared_ptr<Molecule> mol, Options &options,
+SharedMatrix fd_freq_0(std::shared_ptr<Molecule> mol, std::shared_ptr<Wavefunction> wfn, Options &options,
                       const py::list& energies, int irrep=-1);
-SharedMatrix fd_freq_1(std::shared_ptr<Molecule> mol, Options &options,
+SharedMatrix fd_freq_1(std::shared_ptr<Molecule> mol, std::shared_ptr<Wavefunction> wfn, Options &options,
                       const py::list& E_list, int irrep=-1);
```

Comment:
We probably do not want to add additional args to this. Instead have the `fd_freq_1` function return a tuple of `(Hessian, freqs)`.